buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -115,6 +115,11 @@ class PluginBuildPlugin implements Plugin<Project> {
             if (isModule == false || isXPackModule) {
                 addNoticeGeneration(project, extension1)
             }
+            if(project.pluginManager.hasPlugin("elasticsearch.yaml-rest-test")
```

Comment:
nit: space after `if`
However, should the integTest task be disabled through onlyIf? There is no guarantee the plugins checked for here will be applied before esplugin.

Reply:
fixed nit. 
> There is no guarantee the plugins checked for here will be applied before esplugin.

This is part of an `project.afterEvaluate` block, so I think it is guaranteed to be applied.